src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +205,30 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
-}
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A select is used to allow users to pick among a number of options
+	 *   composition: >
+	 *     Select field will render a select-tag with a number of options.
+	 *   effect: >
+	 *     Only one option is selectable.
+	 *
```

Comment:
State some rivals here. I think at least "Checkbox" is one, since Select imo should not be used for a binary yes/no choice. This is Checkboxes domain. I would also like to see that in the rules.